core/tests/protractor_utils/AdminPage.js

```diff
@@ -403,6 +415,8 @@ var AdminPage = function() {
   this.expectUserToBeQuestionReviewer = async function(username) {
     var reviewRight = await _getUserReviewRightsElement(
       username, REVIEW_CATEGORY_QUESTION);
+    await waitFor.visibilityOf(
+      reviewRight, 'Allowed taking too long to appear');
```

Comment:
Can you make this error message more descriptive. We want it to be easy for developers to find what element in the code is not appearing, so maybe `reviewRight` instead of `Allowed`?